src/Authentication/Authentication/Cmdlets/InvokeGraphRequest.cs

```diff
@@ -0,0 +1,965 @@
+                    "WebCmdletBodyConflictException");
+                ThrowTerminatingError(error);
+            }
+
+            // Ensure InFile is an Existing Item
+            if (InFile != null)
+            {
+                ErrorRecord errorRecord = null;
+
+                try
+                {
+                    var providerPaths = GetResolvedProviderPathFromPSPath(InFile, out var provider);
+
+                    if (!provider.Name.Equals(FileSystemProvider.ProviderName, StringComparison.OrdinalIgnoreCase))
+                    {
+                        errorRecord = GetValidationError("NotFilesystemPath",
+                            "WebCmdletInFileNotFilesystemPathException", InFile);
+                    }
+                    else
+                    {
+                        if (providerPaths.Count > 1)
+                        {
+                            errorRecord = GetValidationError("MultiplePathsResolved",
+                                "WebCmdletInFileMultiplePathsResolvedException", InFile);
+                        }
+                        else if (providerPaths.Count == 0)
+                        {
+                            errorRecord = GetValidationError("NoPathResolved",
+                                "WebCmdletInFileNoPathResolvedException", InFile);
+                        }
+                        else
+                        {
+                            if (Directory.Exists(providerPaths[0]))
+                                errorRecord = GetValidationError("DirectoryPathSpecified",
+                                    "WebCmdletInFileNotFilePathException", InFile);
+
+                            _originalFilePath = InFile;
+                            InFile = providerPaths[0];
+                        }
+                    }
+                }
+                catch (ItemNotFoundException pathNotFound)
+                {
+                    errorRecord = new ErrorRecord(pathNotFound.ErrorRecord, pathNotFound);
+                }
+                catch (ProviderNotFoundException providerNotFound)
+                {
+                    errorRecord = new ErrorRecord(providerNotFound.ErrorRecord, providerNotFound);
+                }
+                catch (DriveNotFoundException driveNotFound)
+                {
+                    errorRecord = new ErrorRecord(driveNotFound.ErrorRecord, driveNotFound);
+                }
+
+                if (errorRecord != null) ThrowTerminatingError(errorRecord);
+            }
+        }
+
+        private ErrorRecord GetValidationError(string msg, string errorId)
+        {
+            var ex = new ValidationMetadataException(msg);
+            var error = new ErrorRecord(ex, errorId, ErrorCategory.InvalidArgument, this);
+            return error;
+        }
+
+        private ErrorRecord GetValidationError(string msg, string errorId, params object[] args)
+        {
+            msg = string.Format(CultureInfo.InvariantCulture, msg, args);
```

Comment:
Q: what is the reasoning behind using InvariantCulture here and CurrentCulture in other places?

Reply:
Corrected. Using current culture now consistently.